src/include/primer/primer_hash.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <memory>
+#include <sstream>
+#include <string>
+
+#include "murmur3/MurmurHash3.h"
+
+namespace bustub {
+
+template <typename KeyType>
+class PrimerHashFunction {
```

Comment:
Can we get rid of this and just use `HashUtil`?
https://github.com/cmu-db/bustub/blob/master/src/include/common/util/hash_util.h

Reply:
Done